BookingSystem/BookingSystem.Commands/Commands/HotelCommands/Commands/EditHotelExtraServicesCommand.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using AutoMapper;
+using BookingSystem.Commands.Commands.HotelCommands.DTOs;
+using BookingSystem.Commands.Infrastructure;
+using BookingSystem.Common.Interfaces;
+using BookingSystem.WritePersistence;
+using BookingSystem.WritePersistence.WriteModels;
+
+namespace BookingSystem.Commands.Commands.HotelCommands.Commands
+{
+    public class EditHotelExtraServicesCommand : ICommand<Result>
```

Comment:
where are you going to use this command taking into account you already have add/edit/delete extra service commands?